x-pack/solutions/observability/plugins/slo/public/pages/slo_details/components/external_link_display_text.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export function ExternalLinkDisplayText({
+  textSize,
+  id,
+  url,
+}: {
+  textSize: 's' | 'xs';
+  id: string;
```

Comment:
I think `content` would be a more appropriate name here for this property, you're using this component also to display the slo name.

```suggestion
  content: string;
```